src/types/events/webrtc/ready.rs

```diff
@@ -0,0 +1,29 @@
+use std::net::Ipv4Addr;
+
+use crate::types::WebSocketEvent;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Deserialize, Serialize, Clone)]
+/// The ready event for the webrtc stream;
+/// Used to give info after the identify event;
+/// See https://discord.com/developers/docs/topics/voice-connections#establishing-a-voice-websocket-connection-example-voice-ready-payload;
+pub struct VoiceReady {
+    ssrc: u8,
+    ip: Ipv4Addr,
+    port: u8,
```

Comment:
I don't think `u8` will suffice here, `port` can be in the 10.000 order of magnitude, the linked example even shows a port of '1234'